pkg/background/mutate/mutate.go

```diff
@@ -84,11 +84,27 @@ func (c *MutateExistingController) ProcessUR(ur *kyvernov1beta1.UpdateRequest) e
 			continue
 		}
 
-		trigger, err := common.GetResource(c.client, ur.Spec, c.log)
-		if err != nil || trigger == nil {
-			logger.WithName(rule.Name).Error(err, "failed to get trigger resource")
-			errs = append(errs, err)
-			continue
+		var trigger *unstructured.Unstructured
+		admissionRequest := ur.Spec.Context.AdmissionRequestInfo.AdmissionRequest
+		if admissionRequest != nil && admissionRequest.SubResource != "" {
+			trigger, err = common.GetResource(c.client, ur.Spec, c.log)
+			if err != nil || trigger == nil {
+				logger.WithName(rule.Name).Info("trigger resource not found for subresource, reverting to resource in AdmissionReviewRequest", "subresource", admissionRequest.SubResource)
+				triggerBytes := admissionRequest.Object.Raw
```

Comment:
Do we handle DELETE requests for mutate existing policies? If so, `admissionRequest.Object.Raw` could be empty on deletion.

Reply:
Done.